app/models/meter.rb

```diff
@@ -33,20 +38,32 @@ class Meter < ApplicationRecord
   scope :inactive, -> { where(active: false) }
   scope :no_amr_validated_readings, -> { left_outer_joins(:amr_validated_readings).where(amr_validated_readings: { meter_id: nil }) }
 
-  enum meter_type: [:electricity, :gas]
+
+  CREATABLE_METERS = [:electricity, :gas].freeze
+  PSEUDO_METERS = [:solar_pv, :exported_solar_pv, :electricity].freeze
+  METER_TYPES = (CREATABLE_METERS + PSEUDO_METERS).uniq
+
+  enum meter_type: METER_TYPES
```

Comment:
I'm not 100% keen on this, but it does the trick - we have normal electricity meters and pseudo electricity meters.

Reply:
CREATABLE_METER_TYPES = [🔌 , 😲 ]

METER_TYPES = explicit